### DISCUSSION TITLE
Migrating an existing library to beets

### DISCUSSION THREAD

**Discussion by Stampede** [beetbox/beets#4718] [Q&A] [answered]
I spent several hours with the documentation and tagging on a small sample of my library yesterday. I _think_ I have the configurations figured out enough, but want to run it by some people here before I start going hog wild on my music collection.

First off, one thing that I noticed, is that I imported some new music into beets, and then fed the beets-processed files into Picard, just to see if both programs agreed. Picard seems to want to add more tags than Beets did. Example: On one track, Picard wants to add a tag for "Performer [lead vocals]" and "Original Year" to a track, but Beets left those fields empty. Is there a way to tell Beets to make more tags, based on what it finds from MusicBrainz?

2nd question: I have an existing library and I want to use Beets to update and fix the metadata tagging. I need to make absolutely sure that Beets does not move any files when it does this. I do have a preferred way to organize the music folder structure, but if an existing file is in the "wrong" place, I need it to stay there and only update the metadata tags.
This should do the trick in `config.yaml`??
```
import:
    move: no
    copy: no
```
Then if I want to import **new** music, I could tell beets to respect my directory structure that I have in the `paths:` part of the .yaml with `$ beet import -m NEWMUSICPATH`. Do I have that right?

Thanks for any suggestions or corrections!

**Comment by arsaboo** [comment, marked as answer]
Not all tags are used in beets. For example, `performer` is explicitly [not used](https://github.com/beetbox/beets/blob/master/test/test_player.py#L969) in beets. 

For the second part, you have it right. If you say `no` to both `copy` and `move` in the config file, beets will not move any files during import unless you explicitly ask beets to move using the `-m` flag. You can always do a dry run using the `-p` (pretend) flag to see what will be done.
